yaks/spec/json/confucius.json_api.json

```diff
@@ -1,38 +1,18 @@
 {
-  "meta": {
-    "page": {
-      "limit": 10,
-      "offset": 0,
-      "total": 2
+  "data": {
+    "id": 9,
```

Comment:
This is not really related to your PR itself but JSON API ids must be string (http://jsonapi.org/format/#document-structure-resource-ids). I think we should create a task for this.